parcelgen-runtime/src/com/yelp/parcelgen/JsonUtil.java

```diff
@@ -28,7 +28,7 @@ private JsonUtil() {
      * @return An ArrayList of new objects created by <code>creator</code>.
      * @throws JSONException If parsing of one of the objects fails.
      */
-    public static <E> ArrayList<E> parseJsonList(JSONArray array, JsonParser<E> creator)
+    public static <E extends Parcelable> ArrayList<E> parseJsonList(JSONArray array, JsonParser<E> creator)
```

Comment:
could you standardize this file on the generic name? (sometimes `E` sometimes `T`)